python/notebooks/rendering_urdf.ipynb

```diff
@@ -49,62 +47,62 @@
     "import pythreejs\n",
     "import gtsam\n",
     "from IPython.display import display"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
```

Comment:
not a request for changing anything, but just generally I wonder if this is a jupyter notebook version thing or if generally there is a better way to version control notebooks?

Reply:
There actually is a way, but I don't have the time to figure it out right now.

Here's some help: https://zhauniarovich.com/post/2020/2020-06-clearing-jupyter-output/